Gruntfile.js

```diff
@@ -29,18 +29,18 @@ module.exports = function(grunt) {
         pkg: packageInfo,
 
         jshint: {
+            options: {
+                jshintrc: '.jshintrc'
+            },
             player : [
-                'src/js/**/*.js'
+                'src/js/{,*/}*.js'
```

Comment:
I double checked, and it turns out this ** symbol is called a *globstar* which actually will match files in the same directory. I tested it using node

> glob = require('glob')
> glob('src/js/**/*.js', function(err, files) { console.log(files); });